frontend/src/components/contexts/contexts.ts

```diff
@@ -0,0 +1,11 @@
+import * as React from "react";
+import { MockProfileBackendService } from "../../services/mock_profile_backend";
+import { ProfileHandlerService } from "../../services/profile_handler_service";
+
+const NUM_PROFILES = 10;
+
+export const MockBackendService = new MockProfileBackendService(NUM_PROFILES);
```

Comment:
Add a TODO comment for yourself to change this to a real backend service implementation